x-pack/plugins/security_solution/common/api/timeline/resolve_timeline/resolve_timeline_route_schema.yaml

```diff
@@ -39,9 +39,9 @@ paths:
                     type: object
                     properties:
                       getOneTimeline:
-                        oneOf:
```

Comment:
@janmonschke this is why the handlebars template needed to be updated. The old pattern here wasn't valid in the 3.0.0 schema. We'll need to change it if we update it if we ever change our schema